docs/learn/learn-offenses.md

```diff
@@ -0,0 +1,226 @@
+| Kusama   | 3329 | Small Equivocation | The validator operated a test machine with cloned keys; the test machine was online at the same time as the primary, which resulted in a slash. Details can be found [here](https://kusama.polkassembly.io/post/1343).                                                                                                        | The validator requested a slash cancellation, but the council declined.                                                            |
+| Kusama   | 3995 | Small Equivocation | The validator noticed several errors, after which the client crashed, and a slash was applied. The validator recorded all events and opened GitHub issues to allow for technical opinions to be shared. Details can be found [here](https://kusama.polkassembly.io/post/1733).                                                | The validator requested to cancel the slash. The council approved the request as they believed the error was not operator-related. |
+
+#### Slashing Across Eras
+
+There are three main difficulties to account for with slashing in NPoS:
+
+- A nominator can nominate multiple validators and be slashed via any of them.
+- Until slashed, the stake is reused from era to era. Nominating with N coins for E eras in a row
+  does not mean you have N\*E coins to be slashed - you've only ever had N.
+- Slashable offenses can be found after the fact and out of order.
+
+To balance this, we only slash for the maximum slash a participant can receive in some time period
+rather than the sum. This ensures protection from overslashing. Likewise, the period over which
+maximum slashes are computed is finite, and the validator is chilled with nominations withdrawn
+after a slashing event, as stated in the previous section. This prevents rage-quit attacks in which,
+once caught misbehaving, a participant deliberately misbehaves more because their slashing amount is
+already maxed out.
+
+### Disabling
+
+**Disabling** stops validators from performing specific actions. Disabling is further divided into:
+
+- On-chain disabling lasts for a whole era and stops validators from block authoring, backing
+  through runtime filtering, initiating a dispute, and backing. It also makes other nodes ignore
+  backing statements.
+- Off-chain disabling lasts for a session, is caused by losing a dispute, and stops validators from
+  initiating a dispute.
+
+Off-chain disabling is always a lower priority than on-chain disabling. Off-chain disabling
+prioritizes disabling first backers, then ForInvalid, then AgainstValid.
+
+### Rep
+
+**TODO**
```

Comment:
Some minor offences often connected to spamming are only punished by Networking Reputation Changes. When validators connect to each other they use a reputation metric for each of their peers. If our peers provide valuable data and behave properly we add reputation and if they provide us with faulty or spam data we reduce their reputation. If a validator looses enough reputation the peer will temporarily close the channel. This helps in fighting against DoS (Denial of Service) attacks. The consequences of closing the channel might be varied but in general performing validator tasks will be harder so it might result in lower validators rewards.

Reply:
added